src/core/library.c

```diff
@@ -1352,7 +1352,8 @@ QuicLibraryGetWorker(
     QUIC_DBG_ASSERT(MsQuicLib.WorkerPool != NULL);
     return
         &MsQuicLib.WorkerPool->Workers[
-            MsQuicLib.NextWorkerIndex++ % MsQuicLib.WorkerPool->WorkerCount];
```

Comment:
Why? Just let this roll over?

Reply:
Sanitizer warns when the `uint8_t` exceeds 255. Alternative is to suppress that warning?

Reply:
I think that'd be my preference in this particular case.